<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="create_maze">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>13</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>构建迷宫</string>
   </property>
   <widget class="QSpinBox" name="row_spin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="col_spin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="row_label">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>24</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>行数</string>
    </property>
   </widget>
   <widget class="QLabel" name="col_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>53</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>列数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ensure1_btn">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>115</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/love.jpg</normaloff>:/image/love.jpg</iconset>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="col_label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>84</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>随机</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="random_spin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.200000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>210</y>
     <width>131</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>选择算法</string>
   </property>
   <widget class="QPushButton" name="dfs_stack_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>9</pointsize>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>dfs_stack    </string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/love.jpg</normaloff>:/image/love.jpg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="dfs_recursion_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>dfs_recursion</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/love.jpg</normaloff>:/image/love.jpg</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bfs_queue_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>bfs_queue    </string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/love.jpg</normaloff>:/image/love.jpg</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>430</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>执行详情</string>
   </property>
   <widget class="QLabel" name="path_length_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>路径长度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="min_step">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>29</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="execute_time_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>执行时间</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="run_time">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>70</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adj_matrix_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>9</pointsize>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>邻接矩阵</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/love.jpg</normaloff>:/image/love.jpg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="adj_list_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>9</pointsize>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>邻接表  </string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/love.jpg</normaloff>:/image/love.jpg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="min_path_btn">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>104</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>9</pointsize>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>最短路</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/love.jpg</normaloff>:/image/love.jpg</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
